CDP4WebServices.API/Services/Operations/OperationProcessor.cs

```diff
@@ -89,9 +89,9 @@ public class OperationProcessor : IOperationProcessor
         private readonly string[] topContainerTypes = { "SiteDirectory", "EngineeringModel" };
 
         /// <summary>
-        /// The operation <see cref="Thing"/> instance cache.
+        /// Gets the operation <see cref="Thing"/> instance cache.
         /// </summary>
-        private readonly Dictionary<DtoInfo, DtoResolveHelper> operationThingCache = new Dictionary<DtoInfo, DtoResolveHelper>();
+        public Dictionary<DtoInfo, DtoResolveHelper> OperationThingCache { get; } = new ();
```

Comment:
this is a major change. Please add more docs here to state how to use this now open cache. This was available to the operation processor itself, and now it can be accessed from anywhere. Make it clear for what purpose this cache should be used by outside callers. Please update the interface and class docs

Also, i see some C# version 9 features being used. i don;t have those, so it does not build for me. Do we want to also use C# 9 syntactic sugar?

Reply:
I think we changed all project settings to use the latest C# version a while ago. At least for the IME we did.
So I think you should update VS2019.

Reply:
ok